package.json

```diff
@@ -116,6 +113,7 @@
         }
     },
     "dependencies": {
+        "@babel/polyfill": "^7.12.1",
```

Comment:
This has been removed as a devDependency and the package is deprecated, should this be added here/could we change to the recommended alternative: https://babeljs.io/docs/en/babel-polyfill

Reply:
I can't quite remember why I brought it in to begin with. I've removed it and rerun the tests / rebuilt the project and had a brief play with the app. All seems to work with this removed.